portfolio/src/main/webapp/visualizer.js

```diff
@@ -5,24 +5,24 @@ google.charts.load('current', {
 google.charts.setOnLoadCallback(initRegionsMap);
 
 // holds json on covid cases from fetch request. Holds date and cases by state
-let global_data = []; 
+let global_covid_data = []; 
 
 function initRegionsMap() {
   fetch('/covid-data').then(response => response.json()).then((covidData) => {
     for (let i = 0; i < covidData.length; i++) {
-      global_data[i] = covidData[i];
+      global_covid_data[i] = covidData[i];
     }
     // draw map with inital data 
     drawMap(0); 
   });
 }
 
 /*
-* Draws map at date where date is index of global_data array
+* Draws map at date where index is index of global_data array
 * Index 0 corresponds with 2020-01-21
 */
-function drawMap(date) {
-  if (date >= global_data.length) {
+function drawMap(index) {
+  if (index >= global_covid_data.length || index < 0) {
```

Comment:
I think it might be a better user experience if we never let the `index` value dip below 0. Otherwise if they click it to many times they'll need to click it back that many times to get back to the data.